### PR TITLE
Remove ResultSet

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #161

### CODE REVIEW

**Comment by xepozz** [comment]
What's the purpose?

**Comment by arogachev** [comment]
@xepozz We decided that it's redundant after #151, because now every error contains value path and knows about attribute. Initially that was @vjik's suggestion.

File: src/PostValidationHookInterface.php [hunk 1]
```diff
@@ -11,5 +11,8 @@
  */
 interface PostValidationHookInterface extends DataSetInterface
 {
-    public function processValidationResult(ResultSet $resultSet): void;
+    /**
+     * @param Result $result
+     */
```
Comment (vjik) [inline src/PostValidationHookInterface.php:16]:
```suggestion
```

This is redundant.

**Review by vjik — APPROVED** [review]

File: src/PostValidationHookInterface.php (hunk 1 above)
Comment (arogachev) [reply src/PostValidationHookInterface.php]: Agree. Thanks, fixed. :+1:

Comment (arogachev) [reply src/PostValidationHookInterface.php]: :heavy_check_mark:

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**